src/plugins/hider.ts

```diff
@@ -40,14 +40,13 @@ export class Hider {
 
 export async function processRequest(host: Host) {
     const debug = await host.GetValue('debug') || false;
-    const extensionMode = await host.GetValue('extension-mode');
```

Comment:
>extension-mode [](start = 47, length = 14)

Better to define in ArgumentConstants in model.ts as a const string.

Reply:
I removed this line. defined in merger.